diem-unified-charter.md

```diff
@@ -12,47 +9,44 @@ ISO 7010 defines a set of emblems that can be used to identify hazards, these in
 Today often these emblems/symbols require a sense of sight, or touch to become known to the receiver.
 
 There is a need to sense emblems/symbols through digital communication channels.
-The work of this group therefore focuses on digital emblems that extend the range of identifying marks from the physical (visual and tactile) to the digital realm.
+
+Digital emblems extend the range of identifying marks from the physical (visual and tactile) to the digital realm.
 The presence of a digital emblem represents a new signal available to cyber operators. 
 
-The DIEM WG will define an architecture, set of emblem types, and presentation methods for creating, displaying, validating and interpreting emblems across applications and platforms in a cohesive way.
-The initial scope of the working group will assume that an emblem is also discovered through digital communication, however, ideally the methods defined for emblem creation and validation should not depend on this assumption.
+The DIEM WG will define an architecture and discovery mechanism enabling digital emblems to be presented and validated across applications and platforms in a cohesive way.
 
 # Architectural Consideration
 
-"To bear an emblem" means to present or display a symbol, figure, or device as an identifying mark. 
+"To issue or produce" an emblem, is to design the identifying mark and its associated attributes.
+The emblem issuer's role is to ensure that the identifying mark is easy to recognize and well understood.
```

Comment:
not sure this sentence is useful. Also the issues, as it was written above, is rather then one that asserts certain attributes to an asset by providing the (instant of an) emblem....? the standards body defining the emblem needs to ensure it's well recognisable.

Reply:
let me see if I can eliminate the emblem issuer from the charter text